### PR TITLE
Implementation / Constructive: (10*5=50)

### PR DESCRIPTION
**Pull request by kunaltelangi**
Q1,Q2,Q3
solved
@ganimtron-10 : before commits got messed up so requested new PR

### CODE REVIEW

File: Implementation  Constructive/1.cpp [hunk 1]
```diff
@@ -0,0 +1,18 @@
...
+    int x = 0;
+    while(t--){
+    string a;
+    cin>>a;
+    if(a == "X++" || a == "++X")x++;
+    else x--;
+    }
+    cout<<x<<endl;
+    return 0;
+}
```
Comment (ganimtron-10) [inline Implementation  Constructive/1.cpp]: Add input-output in comments as done in the Strivers_A2Z_Sheet repo

File: Implementation  Constructive/2.cpp [hunk 2]
```diff
@@ -0,0 +1,22 @@
...
+        if(s[i]>'4')
+            s[i]='9'-s[i]+'0';
+    }
+    cout<<s<<endl;
+}
+ 
+int main(){
+    solve();
+    return 0;
+}
```
Comment (ganimtron-10) [inline Implementation  Constructive/2.cpp]: same as above

File: Implementation  Constructive/4.cpp [hunk 3]
```diff
@@ -0,0 +1,11 @@
...
+
+#include <bits/stdc++.h> 
+using namespace std; 
+int main()
+{
+    int n, k, l, c, d, p, nl, np;
+    cin >> n >> k >> l >> c >> d >> p >> nl >> np;
+    cout << min(min(k * l / nl, c * d), p / np) / n << endl;
+    return 0;
+}
```
Comment (ganimtron-10) [inline Implementation  Constructive/4.cpp]: same as above

**Review by ganimtron-10 — CHANGES_REQUESTED** [review]
Hey @kunaltelangi ,
Thanks for doing this!
Could you see the below comments and give some recognizable names to the files instead of `1.cpp`, etc.

**Comment by ganimtron-10** [comment]
To avoid these commit issues and all, please try creating different branches and working with them.
Mostly never work or push the test or other things on the main/master branch.

**Comment by kunaltelangi** [comment]
> To avoid these commit issues and all, please try creating different branches and working with them. Mostly never work or push the test or other things on the main/master branch.

point noted

**Pull request closed without merging**